torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -150,6 +141,33 @@ AOTITorchError aoti_torch_empty_strided(
   });
 }
 
+AOTITorchError aoti_torch_create_tensor_from_blob(
+    AtenTensorHandle* ret,
+    void* data,
+    int64_t ndim,
+    const int64_t* sizes_ptr,
+    const int64_t* strides_ptr,
+    int64_t storage_offset,
+    int32_t dtype,
+    int32_t device_type,
+    int32_t device_index) {
+  AOTI_TORCH_CONVERT_EXCEPTION_TO_ERROR_CODE({
+    c10::IntArrayRef sizes(sizes_ptr, ndim);
+    c10::IntArrayRef strides(strides_ptr, ndim);
+    c10::Device device{
+        static_cast<c10::DeviceType>(device_type),
+        static_cast<c10::DeviceIndex>(device_index)};
```

Comment:
Note that this may break the cpu path as the formed device like ```cpu:0``` is not valid. We may need something similar to P832206536